service/history/configs/task.go

```diff
@@ -30,11 +30,17 @@ import (
 	"go.temporal.io/server/common/tasks"
 )
 
-var DefaultTaskPriorityWeight = map[tasks.Priority]int{
-	tasks.PriorityHigh:   10,
-	tasks.PriorityMedium: 2,
-	tasks.PriorityLow:    1,
-}
+var (
+	DefaultActiveTaskPriorityWeight = map[tasks.Priority]int{
+		tasks.PriorityHigh: 10,
+		tasks.PriorityLow:  1,
+	}
+
+	DefaultStandbyTaskPriorityWeight = map[tasks.Priority]int{
+		tasks.PriorityHigh: 2,
+		tasks.PriorityLow:  1,
```

Comment:
This essentially increased the chance to execute low priority task.
what is the reason behind this? why not use the same priorities as DefaultActiveTaskPriorityWeight?

Reply:
Because standby tasks ideally should use less resource than active tasks. 

The weight is basically
  active ns: 10
  passive ns: 2
  selected task types (deletion tasks), regardless of activeness: 1
which is very close to what we have before.

Previously standby tasks are marked as low priority so it's weight it only 1, instead of 2. And deletion tasks are medium priority so they have weight 2. Basically reversed. I can change them back though.

Reply:
Synced offline. Updated the high priority weight for standby namespace to be 1 as well.